### ISSUE TITLE
data split location

### ISSUE THREAD

**Issue by henryha2019** [UBC-MDS/dsci_522_group3#26]
Before validating correlations, we should ensure the dataset is split into separate subsets to avoid bias and achieve reliable results. We should use the training set to identify initial correlations. This approach helps ensure that the correlations we observe are not specific to one part of the dataset but are generalizable. Finally, we should reserve a test set for the final evaluation to verify that the validated correlations remain consistent and robust.

**Comment by meagangardner** [comment]
Closing this issue as it was resolved - Thank you Henry!
